microcosm_eventsource/tests/ddl/test_migrations.py

```diff
@@ -397,3 +402,86 @@ def test_edge_case_proc_events_delete_with_no_parent_id_constraint(self):
                 parent_id=None,
             ),
         ))
+
+    def test_proc_events_create(self):
+        """
+        Let's say we want to insert a revised event before the started event.
```

Comment:
Can we also have two more tests:
* inject events in the middle
* add events to the end

Reply:
I don't quite follow what you mean by "in the middle" here. The event being added here is in between a few other events. I will def add a test adding something to the end though.

Reply:
Right. Just add tests with new "first"